tests/inputs/mvs_config.json

```diff
@@ -742,7 +742,8 @@
         "project_id": "1",
         "project_name": "Borg Havn",
         "scenario_id": "1",
-        "scenario_name": "Warehouse 14"
+        "scenario_name": "Warehouse 14",
+        "scenario_description": "Main features: Demand of harbour and shore power. Connection to National grid (0.54 NOK/kWh electricity price, 60 NOK/kW monthly peak demand pricing, 0.54 NOK/kWh*0.8 feedin tariff. PV system of 300 kWp. Objective: Optimize battery capacity. Outcomes: Capacity battery and utilization profile. Consumption from national grid. Feed-in into national grid. Outlook: With this, it can be compared whether or not the battery reduced the peak demand expenditures. Later, the simulation results should be compared to the consumption from national grid without batteries."
```

Comment:
Where did you get this long description from? Is it accurate for the specific input file?